### ISSUE TITLE
A Question About Training Loss

### ISSUE THREAD

**Issue by johannwyh** [AlgoHunt/Face-Xray#4]
Hi, Lingzhi !

I have a question about your model training loss. As is stated in your paper, you sum up the batch loss to get a loss number. 

My question is that, did you divide the loss by the batch size? If not, does it mean that my learning rate should be inversely proportional to the batch size?

Thanks a lot! By the way, FaceShifter is awesome and hope we can surpass it!
